### PR TITLE
daos-7503 documentation: csum review and telem section review

### PR DESCRIPTION
**Pull request by cdurf1**
Skip-build: true
Skip-test: true

Signed-off-by: craig <craig.durfey@intel.com>

### CODE REVIEW

**Review by ryon-jensen — COMMENTED** [review]
Looks good to me, just one small nit. A little confused about the title though. These changes don't appear to be related to checksums?

File: src/rebuild/README.md [hunk 1]
```diff
@@ -32,9 +32,9 @@ The rebuild is divided into 2 phases, scan and pull.
 
 Initially, the leader will propagate the failure notification
 to all other surviving targets by a collective RPC. Any target
-that receives this RPC will start to scan its own object table
-to determine the object losts data redundancy on the faulty
-target, if it does, then send their IDs and related metadata
+that receives this RPC will start to scan its object table
+to determine the objects lost data redundancy on the faulty
+target If it does, send their IDs and related metadata
```
[the remark this answers is not in the document]
Comment (johannlombardi) [reply src/rebuild/README.md:37]: To be addressed.

**Comment by johannlombardi** [comment]
any update?

**Comment by cdurf1** [comment]
@johannlombardi  - README.md issue fixed. 
@ryon-jensen  - I placed it under the same PR title as I was unable to get the review complete before it was landed

**Review by ryon-jensen — APPROVED** [review]

**Review by johannlombardi — APPROVED** [review]

**Pull request merged**